xgbse/converters.py

```diff
@@ -112,3 +112,16 @@ def build_xgb_cox_dmatrix(X, T, E):
     target = np.where(E, T, -T)
 
     return xgb.DMatrix(X, label=target)
+
+
+def to_survival(interval):
```

Comment:
What do you think of calling this function `hazard_to_survival`? I think itd be more intuitive, although this is described in the docstring. Think of someone reading a code that have a call to this function. That someone may not need to open the function to see what it really converts into a survival, if its name is more intuitive.